backports/py-opengl/APKBUILD

```diff
@@ -15,6 +15,9 @@ install=""
 subpackages=""
 source="https://files.pythonhosted.org/packages/source/P/PyOpenGL/$_pkgname-$pkgver.tar.gz"
 
+replaces="py-opengl" # Backwards compatibility
+provides="py-opengl=$pkgver-r$pkgrel" # Backwards compatibility
```

Comment:
It makes `py-opengl` ailias of `py2-opengl`.
Alpine 3.11 (ROS Noetic) has native `py3-opengl`.